Resources/Textures/_Harmony/Clothing/Hands/Gloves/hop.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Icon and inhand drawn by Ubaser and takem from this atcommithttps://github.com/space-wizards/space-station-14/pull/21915 , equipped sprite made by CrimsonJupiter for use in Harmony",
```

Comment:
* Multiple typos in copyright.
* `taken from this` from what?
* `at commit <pr link>` PR is not a commit.

Reply:
> * Multiple typos in copyright.
> * `taken from this` from what?
> * `at commit <pr link>` PR is not a commit.

Every single mention of a pull has been replaced with a commit in the "at commit <link>" format that you used. I hope this works!